src/test/scala/com/linagora/openpaas/gatling/CalendarAndContactsWithRefreshTokenSimulation.scala

```diff
@@ -21,6 +23,9 @@ class CalendarAndContactsWithRefreshTokenSimulation extends Simulation {
   val eventUuidFeeder: Iterator[Map[String, String]] = Iterator.continually(Map("eventUuid" -> randomUuidString))
   val scn: ScenarioBuilder = scenario("CalendarAndContactsPlatformTestScenario")
     .feed(userFeeder)
+    .exec(login)
+    .exec(getProfile())
+    .exec(logout)
```

Comment:
These are already done in the `openCalendarSPA` & `openContactsSPA` steps.

Reply:
i don't do it here, i have errors reporting that the session key are missing.
I think they are done too in the scenario

Reply:
Thanks, I'll see after the login issue has been fixed.